shortfin/python/shortfin_apps/llm/cli.py

```diff
@@ -26,9 +28,9 @@
 
 def add_input_args(parser):
     group = parser.add_argument_group("Input Source", "Inputs to select from")
-    group = group.add_mutually_exclusive_group()
-    parser.add_argument("--prompt")
-    parser.add_argument("--prompt-file")
+    group = group.add_mutually_exclusive_group(required=True)
+    group.add_argument("--prompt")
+    group.add_argument("--prompt-file")
 
 
 def add_service_args(parser):
```

Comment:
neat. love how this is organized. new trick acquired (for organizing argparse args into groups)